packages/Tabs/src/tabs.scss

```diff
@@ -1,115 +1,212 @@
 }
 
-.ids-tabs {
-  font-family: var(--ids-tabs-font-family);
-  font-size: var(--ids-tabs-font-size);
+[data-brand="workleap"] {
+    /* Default */
+    --ids-tabs-font-family: var(--hop-body-sm-font-family);
+    --ids-tabs-font-size: var(--hop-body-sm-font-size);
+    --ids-tabs-margin: #{tokens.$space-0} #{tokens.$space-0} 1.25rem;
+
+    /* Tab */
+    --ids-tab-button-padding: var(--hop-space-inset-sm);
+    --ids-tab-button-font-size: var(--hop-body-md-font-size);
+    --ids-tab-button-font-weight: var(--hop-body-md-medium-font-weight);
+    --ids-tab-button-line-height: var(--hop-body-md-medium-line-height);
+    --ids-tab-button-color: var(--hop-neutral-text-weak);
+    --ids-tab-button-hover-color: var(--hop-neutral-text-hover);
+    --ids-tab-button-active-color: var(--hop-neutral-text);
+    --ids-tab-button-after-horizontal-space: 0.5rem;
+    --ids-tab-button-after-height: 0.125rem;
+    --ids-tab-button-after-bottom: 0.25rem;
+    --ids-tab-button-after-background: var(--hop-neutral-border-active);
+    --ids-tab-button-focus-outline-size: 0.125rem;
+    --ids-tab-button-focus-outline-color: var(--hop-primary-surface-strong);
+    --ids-tab-button-focus-outline-radius: var(--hop-shape-rounded-sm);
+    --ids-tab-button-focus-height: calc(100% - calc(var(--ids-tab-button-padding) * 2));
+    --ids-tab-button-focus-width: calc(100% - calc(var(--ids-tab-button-padding) * 2 - calc(var(--ids-tab-button-focus-outline-size) * 2)));
+
+    /* Icons */
+    --ids-tab-icon-margin-left: var(--hop-space-inline-xs);
+    --ids-tab-icon-notification-color: var(--hop-primary-surface-strong);
+    --ids-tab-icon-premium-color: var(--hop-upsell-icon-weak);
+
+    /* === Inline Style === */
+
+    --ids-inline-tab-background-color: var(--hop-neutral-surface-weakest);
+    --ids-inline-tab-border-radius: var(--hop-shape-rounded-md);
+    --ids-inline-tab-outline: 0.0625rem solid var(--hop-neutral-border-weak);
+    --ids-inline-tab-outline-offset: -0.0625rem;
+    --ids-inline-tab-outline-focus-offset: -6px;
+    --ids-inline-tab-button-font-size: var(--hop-body-sm-font-size);
+    --ids-inline-tab-button-line-height: var(--hop-body-sm-line-height);
+    --ids-inline-tab-button-focus-height: calc(100% - 1.5rem);
+    --ids-inline-tab-button-focus-width: calc(100% - 1.5rem);
+
+    /* Tab */
+    --ids-inline-tab-button-padding: calc(1rem - 0.125rem) 1rem;
+    --ids-inline-tab-button-after-bottom: 0;
+    --ids-inline-tab-button-after-height: 0.125rem;
+    --ids-inline-tab-button-after-horizontal-space: 1rem;
+}
 
-  &__list {
-    display: flex;
-    list-style: none;
-    margin: var(--ids-tabs-margin);
-    padding: 0;
-  }
+.ids-tabs {
+    font-family: var(--ids-tabs-font-family);
+    font-size: var(--ids-tabs-font-size);
+
+    &__list {
+        display: flex;
+        list-style: none;
+        margin: var(--ids-tabs-margin);
+        padding: 0;
+
+        .ids-tabs--inline &  {
```

Comment:
There's a statement below about inline rendering, so why duplicate it here?

Reply:
notice the & at the end? This declaration is only for links/buttons that are inline where below the declaration is for inline tabs not the buttons inside an inline tab